docs/developer-guide/fetching-data.md

```diff
@@ -13,6 +13,8 @@ The following steps must be completed before you can pull data from an evironmen
 2. On the `#ds-etna` or `#ds-etna-dev` Slack channels, request for someone to add you to the Etna project.
 3. Once you have access, [generate an API token](https://docs.platform.sh/development/cli/api-tokens.html#get-a-token) for your account (The name **Local CLI** will do nicely), and add it your local `.env` file as `PLATFORMSH_CLI_TOKEN`.
```

Comment:
```suggestion
3. Once you have access, [generate an API token](https://docs.platform.sh/development/cli/api-tokens.html#get-a-token)** for your account (The name **Local CLI** will do nicely), and add it your local `.env` file as `PLATFORMSH_CLI_TOKEN`.
```

Reply:
Thanks @janehughes - did you mean to apply the `**`s round a particular bit of text? There's just one set here so not sure exactly which text you meant to highlight.

Reply:
Sorry I meant to use `**` to signal a footnote, though I now realise that's a confusing suggestion given ** also means bold text!

Reply:
Ah ok, in that case I will leave it I think. @janehughes are you happy to approve the MR?